### PR TITLE
Adds adding new terminology section

### PR DESCRIPTION
**Pull request by bcolsson**

### CODE REVIEW

File: src/tools/pontoon/adding_new_terminology.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Add the new term
+
+Find and click `Terms` on the navigation pane on the left. Check on the following page to make sure a term does not already exist by searching in the search field near the top.
+
+To add a new term, click `Add Term +`. The following page will have the following fields:
+
+* Text: Term you wish to register.
+* Entity: *Ignore*
+* Part of speech: Select the part of speech that applies to your term. In some cases the same string can be registered twice with different parts of speech,e.g. bookmark as a noun (“open your bookmarks”) or as a verb (“bookmark this website”).
```
Comment (Delphine) [inline src/tools/pontoon/adding_new_terminology.md:13]: missing space between the comma and e.g.

File: src/tools/pontoon/adding_new_terminology.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Definition: Definition of the meaning of the term, or explanatio of what the term is.
```
Comment (Delphine) [inline src/tools/pontoon/adding_new_terminology.md:14]: nit: explanation

**Review by Delphine — COMMENTED** [review]
Just a couple nits. Otherwise good for me. Thanks!

**Comment by flodolo** [comment]
@bcolsson 
For documentation PRs, it's better to wait for someone else to review them before merging.

You can flag @mozilla-l10n/pm-group for that.

**Pull request merged**